From 9beb3c585cc9c19a3052eddf0f68ef4d29884938 Mon Sep 17 00:00:00 2001
From: Jianming Qiao <jianming.qiao@bp.renesas.com>
Date: Fri, 17 Aug 2018 02:24:20 +0100
Subject: [PATCH] Patching Eigen library on the fly via Bazel

We have to patch Eigen via Bazel instead of BitBake as in the middle
of Bazel running there is no way for BitBake to know that the Eigen
library has been downloaded by Bazel.

Signed-off-by: Jianming Qiao <jianming.qiao@bp.renesas.com>
---
 tensorflow/workspace.bzl | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tensorflow/workspace.bzl b/tensorflow/workspace.bzl
index 378de42..cb785f2 100644
--- a/tensorflow/workspace.bzl
+++ b/tensorflow/workspace.bzl
@@ -114,6 +114,7 @@ def tf_workspace(path_prefix="", tf_repo_name=""):
       ],
       sha256 = "d956415d784fa4e42b6a2a45c32556d6aec9d0a3d8ef48baee2522ab762556a9",
       strip_prefix = "eigen-eigen-fd6845384b86",
+      patch_file = clean_dep("//third_party:Fix-eigen-header-compiler-crash.patch"),
       build_file = clean_dep("//third_party:eigen.BUILD"),
   )
 
-- 
1.9.1

